pkg/api/aim/request/metric.go

```diff
@@ -1,5 +1,26 @@
 package request
 
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+// GetAlignedMetricRequest is a request object for `GET /mlflow/metrics/align` endpoint.
+type GetAlignedMetricRequest struct {
+	AlignBy string                    `json:"align_by"`
+	Runs    []AlignedMetricRunRequest `json:"runs"`
+}
+
+type AlignedMetricRunRequest struct {
+	ID     string                      `json:"run_id"`
+	Traces []AlignedMetricTraceRequest `json:"traces"`
+}
+
+type AlignedMetricTraceRequest struct {
+	Context fiber.Map `json:"context"`
+	Name    string    `json:"name"`
+	Slice   [3]int    `json:"slice"`
```

Comment:
is it really only 3 items size?

Reply:
I'm not really sure, I used 3 since I saw it on `SearchAlignedMetrics` endpoint, but anyway now I changed it.

Reply:
no no, I just curious why only 3?

Reply:
@jgiannuzzi is there a reason to have array of 3 elements?

Reply:
as I can see from the code - not really.